docs/formats.rst

```diff
@@ -101,31 +101,31 @@ Below are listed capabilities of all supported formats.
 
 .. [#m] See :ref:`bimono`
 .. [#p] Plurals are necessary to properly localize strings with variable count.
-.. [#n] Comments can be used to pass additional information about string to translate.
-.. [#c] Context is used to differentiate same strings used in different scope (for example `Sun` can be used as abbreviated name of day or as a name of our closest star).
-.. [#l] Location of string in source code might help skilled translators to figure out how the string is used.
-.. [#a] Additional states supported by the file format in addition to not translated and translated.
-.. [#x] XML comment placed before the ``<string>`` element is parsed as a developer comment.
+.. [#n] Comments can be used to pass additional info about string to translate.
```

Comment:
About ? string to translate.
what, the?

Reply:
the